.travis.yml

```diff
@@ -5,6 +5,7 @@ language: python
 sudo: false
 cache: pip
 python:
+  - "3.7"
```

Comment:
It works this way:
```yml
matrix:
  include:
    - python: 3.7
      dist: xenial
      sudo: required
```

Reply:
@sloria The `matrix` block is correct, but you have to remove line 8 (the one I've commented on).